src/api/spec/controllers/webui/users_controller_spec.rb

```diff
@@ -132,14 +132,8 @@
       it "changes the state to 'deleted'" do
         delete :destroy, params: { login: user.login }
         expect(user.reload.state).to eq('deleted')
-      end
-
-      it 'handles validation errors' do
-        user.update_attributes(email: 'invalid')
-        user.save!(validate: false)
-        delete :destroy, params: { login: user.login }
-        expect(user.reload.state).to eq('confirmed')
-        expect(flash[:error]).to eq("Marking user '#{user.login}' as deleted failed: Email must be a valid email address")
+        expect(user.reload.email).to eq('')
+        expect(user.reload.realname).to eq('')
```

Comment:
You are not testing project deletion anywhere.

Reply:
the mini test does (partly indirect via backend/api sync test)